src/Symfony/Component/DependencyInjection/Tests/Fixtures/GetterOverriding.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Tests\Fixtures;
+
+if (PHP_VERSION_ID >= 70100) {
```

Comment:
this check can be removed:, because it doesn't work anyway: the following will trigger a parse error